helper.go

```diff
@@ -0,0 +1,15 @@
+package testy
+
+import (
+	"fmt"
+)
+
+// TestEach runs tester as a subtest for each value in values. The values should have a good default string
+// representation so the subtest names are legible. (Consider implementing fmt.Stringer for complex structs.)
+func TestEach[V any](t TestingT, values []V, tester func(TestingT, V)) {
```

Comment:
actually I think `TestEach[V any](t TestingT, tester (func TestingT, v), values ...V)` might make more sense. Thoughts?

Reply:
actually no, that makes writing the func inline really ugly

Reply:
oh yeah that would be ugly